src/enums/taskEvent.ts

```diff
@@ -0,0 +1,10 @@
+export enum TaskEvent {
+  TASK_STARTED = 'TASK_STARTED',
+  TASK_PROGRESS = 'TASK_PROGRESS',
```

Comment:
isn't "update" better as it includes progress

Reply:
changed to TASK_UPDATED